iceberg/iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergMigration.java

```diff
@@ -133,6 +135,39 @@ public void testRollbackMigratePartitionedBucketedHiveTableToIceberg() throws TE
     validateMigrationRollback(tableName);
   }
 
+  @Test
+  public void testMigrationFailsForUnsupportedSourceFileFormat() {
+    // enough to test once
+    Assume.assumeTrue(fileFormat == FileFormat.ORC && isVectorized &&
+        testTableType == TestTables.TestTableType.HIVE_CATALOG);
+    String tableName = "tbl_unsupported";
+    List<String> formats = ImmutableList.of("TEXTFILE", "JSONFILE", "RCFILE", "SEQUENCEFILE");
+    formats.forEach(format -> {
+      shell.executeStatement("CREATE EXTERNAL TABLE " +  tableName + " (a int) STORED AS " + format + " " +
+          testTables.locationForCreateTableSQL(TableIdentifier.of("default", tableName)));
+      shell.executeStatement("INSERT INTO " + tableName + " VALUES (1), (2), (3)");
+      AssertHelpers.assertThrows("Migrating a " + format + " table to Iceberg should have thrown an exception.",
+          IllegalArgumentException.class, "Cannot convert hive table to iceberg with input format: ",
+          () -> shell.executeStatement("ALTER TABLE " + tableName + " SET TBLPROPERTIES " +
+              "('storage_handler'='org.apache.iceberg.mr.hive.HiveIcebergStorageHandler')"));
+      shell.executeStatement("DROP TABLE " + tableName);
+    });
+  }
+
+  @Test
+  public void testMigrationFailsForManagedTable() {
+    // enough to test once
+    Assume.assumeTrue(fileFormat == FileFormat.ORC && isVectorized &&
```

Comment:
Do we want to have a test for unsupported fileformat?

Reply:
I think `testMigrationFailsForUnsupportedSourceFileFormat` already provides tests for unsupported source table file formats. Or do you mean having a test where the source table is managed AND its format is unsupported at the same time?

Reply:
Nope - I did not know we have `testMigrationFailsForUnsupportedSourceFileFormat`. Please disregard my comment